test/parallel/xyk-pallet.API.sellAsset.test.ts

```diff
@@ -162,9 +196,9 @@ describe("xyk-pallet - Sell assets tests: SellAsset Errors:", () => {
     remainingOfCurrency1 = testUser1.getAsset(firstCurrency)?.amountAfter.free!;
     expect(remainingOfCurrency1).bnEqual(new BN(1));
 
-    const secondWalletAmount = defaultCurrecyValue
+    const secondWalletAmount = defaultCurrencyValue
       .add(new BN(1))
-      .sub(second_asset_amount.div(new BN(2)))
+      .sub(secondAssetAmount)
```

Comment:
is nto this the /2  of what it was before?

Reply:
No, I changed `secondAssetAmount `value

Reply:
allright.